pkg/envoy/xds_server.go

```diff
@@ -912,6 +912,7 @@ func getListenerFilter(isIngress bool, useOriginalSourceAddr bool, proxyPort uin
 		BpfRoot:                  bpf.BPFFSRoot(),
 		IsL7Lb:                   false,
 		ProxyId:                  uint32(proxyPort),
+		PolicyUpdateWarningLimit: durationpb.New(option.Config.FQDNProxyResponseMaxDelay),
```

Comment:
what about setting the same option in `getBPFMetadataListenerFilter` on `BPFMetadata` in `cec_resource_parser.go`?

Reply:
Done!